File: src/main/java/br/com/rotaractsorocabaleste/rotasales/item/ItemService.java
package br.com.rotaractsorocabaleste.rotasales.item;

import java.util.Set;
import java.util.UUID;

public interface ItemService {

    Item create(final Item item);

    boolean belongsToEvent(final UUID id, final UUID eventId);

    Set<Item> findItemsByEventId(final UUID eventId);
}

File: src/main/kotlin/br/com/rotaractsorocabaleste/rotasales/core/service/ItemService.kt
package br.com.rotaractsorocabaleste.rotasales.core.service

import br.com.rotaractsorocabaleste.rotasales.core.entity.Item
import java.util.*

interface ItemService {
    fun save(item: Item): Item
    fun findByEventId(eventId: UUID): List<Item>
}